buildconfig.env.example

```diff
@@ -1,3 +0,0 @@
-export sauceport=5555
```

Comment:
Exactly reason why I removed it,there is no hidden magic in it that cannot be added when needed.